tiledb/sm/serialization/capnp_utils.h

```diff
@@ -34,17 +34,46 @@
 #define TILEDB_CAPNP_UTILS_H
 
 #ifdef TILEDB_SERIALIZATION
+#include "tiledb/sm/serialization/tiledb-rest.capnp.h"
 
 #include "tiledb/common/heap_memory.h"
 #include "tiledb/common/status.h"
+#include "tiledb/sm/array/array.h"
 #include "tiledb/sm/array_schema/array_schema.h"
 #include "tiledb/sm/array_schema/dimension.h"
 #include "tiledb/sm/buffer/buffer.h"
 #include "tiledb/sm/enums/datatype.h"
-#include "tiledb/sm/serialization/tiledb-rest.capnp.h"
 
 using namespace tiledb::common;
 
+namespace tiledb {
+namespace sm {
+namespace serialization {
+
+#ifdef TILEDB_SERIALIZATION
```

Comment:
Appears bulk of capnp_utils.h is already subject to TILEDB_SERIALIZATION (further 'above'), this #if unnecessary?

Reply:
Removed, thanks